docs/about/releases.mdx

```diff
@@ -14,6 +14,18 @@ description: "Latest new features and changelog."
 ---
 
 [All releases](https://github.com/mage-ai/mage-ai/releases)
+- [0.9.72 | House Targaryen Release](https://github.com/mage-ai/mage-ai/releases/tag/0.9.72)
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/abiOckshzu8"
+  title="X-Men Release"
```

Comment:
@thomaschung408 Don't think this affects anything visible in the UI, but can you update the title here?